### PR TITLE
 added Groupviewmodel and completed views for start group

### PR DESCRIPTION
**Pull request by ericdiazjr21**

### CODE REVIEW

File: group/src/main/java/com/android/group/model/firebase/Group.java [hunk 1]
```diff
@@ -3,15 +3,120 @@
...
-    public Group() {
+    private final String groupName;
+    private final String category;
+    private final String iD;
+    private final List<Object> userList;
+    private final String description;
+    private final int users;
+    private final double lat;
+    private final double lng;
+
```
Comment (Moonianne) [inline group/src/main/java/com/android/group/model/firebase/Group.java:16]: in order for firebase to use this model it needs a default constructor

File: group/src/main/java/com/android/group/viewmodel/GroupViewModel.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+public class GroupViewModel extends ViewModel {
+
+    private static GroupViewModel singleInstance;
+    private Group currentGroup;
+    private static final String TAG = "GroupViewModel";
+
+    private GroupViewModel(){}
+
+    public static GroupViewModel getSingleInstance() {
```
Comment (Moonianne) [inline group/src/main/java/com/android/group/viewmodel/GroupViewModel.java]: why make this viewmodel a singleton? Won't it be subject to the ViewModel Providers?

File: group/src/main/java/com/android/group/viewmodel/NetworkViewModel.java [hunk 3]
```diff
@@ -37,6 +37,8 @@ public void setUserSelectedVenue(Venue venue){
         venueSelectedListener.venueSelected(venue);
     }
 
+
```
Comment (Moonianne) [inline group/src/main/java/com/android/group/viewmodel/NetworkViewModel.java]: empty lines >.<

**Review by Moonianne — CHANGES_REQUESTED** [review]
im going to pull this branch to take a better look, on the surface it looks good minus the nitpicks i commented

File: group/src/main/java/com/android/group/viewmodel/GroupViewModel.java (hunk 2 above)
Comment (ericdiazjr21) [reply group/src/main/java/com/android/group/viewmodel/GroupViewModel.java]: I think you right on this.

File: group/src/main/java/com/android/group/viewmodel/NetworkViewModel.java (hunk 3 above)
Comment (ericdiazjr21) [reply group/src/main/java/com/android/group/viewmodel/NetworkViewModel.java]: lalalalalala

File: group/src/main/java/com/android/group/model/firebase/Group.java (hunk 1 above)
Comment (ericdiazjr21) [reply group/src/main/java/com/android/group/model/firebase/Group.java:16]: ill added it

**Review by Moonianne — APPROVED** [review]

**Pull request merged**